Makefile-man.am

```diff
@@ -2600,16 +2605,6 @@ man/systemd-user-sessions.html: man/systemd-user-sessions.service.html
 
 endif
 
-if HAVE_PYTHON
-MANPAGES += \
-	man/systemd.directives.7 \
-	man/systemd.index.7
-MANPAGES_ALIAS += \
-	#
-
```

Comment:
That's a known bug in how `update-man-list` is called: the man pages must be built before `update-man-list` is called. This shortcoming is fixed in the meson build. I'd just ignore it (but revert this chunk of course) since Makefile.am is supposed to be going away.